server/datastore/mysql/scripts.go

```diff
@@ -748,16 +758,17 @@ func (ds *Datastore) LockHostViaScript(ctx context.Context, request *fleet.HostS
 	(
 		host_id,
 		lock_ref,
-		unlock_ref
```

Comment:
Curious: why remove the `unlock_ref` from the query?

Reply:
It was inserting `unlock_ref` to `NULL`, but that's the default value for a NULLable column, so it doesn't need to be provided.